core/zms/src/main/rdl/SignedDomains.rdli

```diff
@@ -49,11 +49,13 @@ type DomainData Struct {
     String applicationId (optional); // associated application id
 }
 
-//A domain object signed with server's private key 
+//A domain object signed with server's private key. The signature
+//and keyid are optional if the metaonly flag is set to true in the
```

Comment:
They're never present when metaonly is true, correct?

Reply:
yes, if metonly is true, they're never present.